config/samples/config.yaml

```diff
@@ -0,0 +1,26 @@
+clusterName: "example-cluster"
+timeIntervalSec: 60
+repoCredentials:
+  "key1":
+    HTTPCredential:
+      Username: "exampleUser"
+      Password:
+        name: "http-creds"
+        key: "password"
+  "key2":
+    SSHCredential:
+      SSHKey:
+        name: "ssh-creds"
+        key: "sshKey"
+  "key3":
+    TLS:
+      insecureSkipVerify: true
```

Comment:
I'm kind of thinking this should be false in your example? I think it means that we don't try to verify the server side's identity. If we have all of those files listed below, then we probably are verifying their identity.

Reply:
Picking this up in follow up pr

Reply:
any reason you don't want to change it here?

Reply:
I know it's a little crazy making for you to be working on these two PRs at the same time. At the same time, I don't want PR comments to get lost, so in general it would be good if we can address within the PR